libimage/manifest_list.go

```diff
@@ -396,6 +396,13 @@ type ManifestListPushOptions struct {
 	Instances []digest.Digest
 	// Add existing instances with requested compression algorithms to manifest list
 	AddCompression []string
+	// Maximum number of retries with exponential backoff when facing
+	// transient network errors.  A reasonable default is used if not set.
+	// Default 3.
+	MaxRetries *uint
+	// RetryDelay used for the exponential back off of MaxRetries.
+	// Default 1 time.Scond.
+	RetryDelay *time.Duration
```

Comment:
Those fields are already included in `CopyOptions`.  Why not use that directly?

Reply:
Can do but `manifest` uses its own struct with fewer/different fields https://github.com/containers/common/blob/main/libimage/manifests/manifests.go#L60 than `CopyOptions` . `CopyOptions` contains lot of other fields with actual `manifest` API is not using.

Reply:
But this structure already includes `CopyOptions`.  Why ship the same fields twice in the same struct?

Reply:
@vrothberg My bad i misunderstood your comment for changes in `libimage/manifests/manifest.go` , its only needed there no here. Removed.